### PR TITLE
Setup sun-ci

### PR DESCRIPTION
**Pull request by hainv-2854**
> > ## Related Tickets
> > 
> > * [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/44862)
> > 
> > ## WHAT (optional)
> > 
> > * Setup Sun*CI

### CODE REVIEW

**Comment by hainv-2854** [comment]
image

**Comment by hainv-2854** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    environment:
+      MYSQL_DATABASE: db_test
+      MYSQL_USER: user_test
+      MYSQL_PASSWORD: password_test
+      MYSQL_ROOT_PASSWORD: password_test
+  before_script:
+  - bundle _2.1.2_ install --path vendor/bundle
+  script:
+  - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+  - bundle _2.1.2_ exec rspec
```
Comment (vanvtt-0952) [inline .sun-ci.yml]: comment line này đi bạn

File: app/models/category.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class Category < ApplicationRecord
-  has_many :products
+  has_many :products, dependent: :destroy
   belongs_to :parent, class_name: Category.name, optional: true
```
Comment (vanvtt-0952) [inline app/models/category.rb]:
```suggestion
  belongs_to :parent, class_name: Category.name, optional: true
  has_many :products, dependent: :destroy
```

**Comment by hainv-2854** [comment]
ready

**Pull request merged**